agent/structs/structs.go

```diff
@@ -1765,6 +1765,11 @@ type IndexedConfigEntries struct {
 	QueryMeta
 }
 
+type IndexedGatewayServices struct {
```

Comment:
You should probably move this in the file so you don't split the struct definition of `IndexedConfigEntries` from it's method set.